SOURCES/etc/xapi.d/plugins/updater.py

```diff
@@ -13,13 +13,13 @@
 from xcpngutils import configure_logging, run_command, error_wrapped
 from xcpngutils.filelocker import FileLocker
 
-
 DEFAULT_REPOS = ('xcp-ng-base', 'xcp-ng-updates')
 
 
 class OperationException(Exception):
     pass
 
+
```

Comment:
The formatting changes are not necessary for the fix, and our pycodestyle configuration does not enforce two lines between functions.